src/args.rs

```diff
@@ -110,9 +110,13 @@ pub struct CliArgs {
     /// Enable file uploading
     #[clap(short = 'u', long = "upload-files")]
     pub file_upload: bool,
+    
+    /// Enable creating directories
+    #[clap(short = 'U', long = "mkdir", requires("file-upload"))]
+    pub mkdir_enabled: bool,
 
     /// Specify uploadable media types
-    #[clap(arg_enum, short = 'm', long = "media-type", requires = "file-upload")]
```

Comment:
Hey, what's the reason for touching this one?

Reply:
Sorry, I have no idea why I touched that.